events/secret_events/wc_plague_secrets_events.txt

```diff
@@ -0,0 +1,463 @@
+﻿#Events for having secrets
+
+namespace = plague_secrets
+
+###MURDER SECRETS###
+
+#I have been exposed as a someone who has Attempted Murder
+plague_secrets.0115 = { #by Mathilda Bjarnehed
```

Comment:
Why do you need these events? Need your answer before I can suggest anything.

Reply:
I copied those from the murder scheme, but because they needed a lot of extra code to make them work (Plaguer trait for instance) and wasn't sure it made sense, I just left those there for now. Basically, if you try to plague/manage to plague someone's lands should it be treated like a murder? like: there's a secret, if the secret is found out you can be blackmailed... I don't think the Plaguer trait makes that much sense, but I'm not sure about the whole secret thing, what do you think?

Reply:
If they're vanilla copied and just need a few changes.
You may delete these and tweak vanilla events to work for both murder and plague:
- Set a scope value to define if it's murder or plague (the scope value will be accessible in the event chain)
- Add text variations (descs, options, etc.) for plague

> I don't think the Plaguer trait makes that much sense, but I'm not sure about the whole secret thing, what do you think?

It depends. If it's used only by Ner'zhul, prob not.

Reply:
> If they're vanilla copied and just need a few changes.

I'll try that, they're almost the same so it shouldn't be hard.


> It depends. If it's used only by Ner'zhul, prob not.

Right now it's just for him so for now I'll keep it simple and won't add any special trait, if later we find out it's a good idea for other characters to do plagues we could add it.